### PR TITLE
Adding security tips for wallet management and node setup

### PR DESCRIPTION
**Pull request by jacklaing**

### CODE REVIEW

File: buy-store-and-stake-pokt.md [hunk 1]
```diff
@@ -44,9 +44,25 @@ Store your Private Key safely. This will be a second method for you to import yo
 Never reveal your Private Key to anyone. If someone has access to your Private Key, they have full access to all the funds. The only application that should ever need access to your Private Key is a wallet.
 {% endhint %}
 
+### Backup Wallet
+
+It is important that you backup your Key File, passphrase, and private key securely. There are two considerations when choosing a backup method:
+* **Theft Prevention**: making sure no-one else can access your Key File, passphrase, or private key
```
Comment (BlueLahar) [inline buy-store-and-stake-pokt.md:50]: I would write: make sure [not "making sure"}

File: buy-store-and-stake-pokt.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ Make sure to store your passphrase safely, if you lose it you will not be able t
 If you accidentally click Back, as long as you downloaded your Key File and saved your passphrase, you will now be able to import your new wallet.
 
 {% hint style="warning" %}
-Make sure to store your Key File safely, if you lose it you will only be able to use the Private Key method to import your wallet anywhere.
+Make sure to store your Key File safely, if you lose it you will only be able to use the Private Key method to import your wallet anywhere. The Key File is the more secure method of importing your wallet since it is encrypted by your passphrase.
```
Comment (BlueLahar) [inline buy-store-and-stake-pokt.md:30]: "anywhere"? do you mean "on any device"? if so, I'd say that as it's clearer

File: buy-store-and-stake-pokt.md [hunk 3]
```diff
@@ -44,9 +44,25 @@ Store your Private Key safely. This will be a second method for you to import yo
...
+* **Theft Prevention**: making sure no-one else can access your Key File, passphrase, or private key
+* **Loss Prevention**: making sure you don't lose access to your Key File, passphrase, or private key
+
+The most secure way to prevent theft is to store your Key File, passphrase, and private key on a USB drive (or other secure external drive), disconnected from any online "cloud" service or internet-connected computer that can be hacked. 
+
+{% hint style="info" %} POKT will soon have Ledger support, which provides a little more advanced functionality than a USB drive, but there is no ETA on this. {% endhint %}
+
+For extra-secure theft prevention, you should split the Key File, passphrase and private key across separate drives and password-protect these drives so that anyone who gets their hands on a drive can't access the contents. For extra loss prevention, you should make multiple copies of these drives and store them in different locations, such as a personal safe, the safe of a trusted family member, and a bank's safety-deposit box.
```
Comment (BlueLahar) [inline buy-store-and-stake-pokt.md:57]: bank safety deposit box

File: buy-store-and-stake-pokt.md [hunk 4]
```diff
@@ -44,9 +44,25 @@ Store your Private Key safely. This will be a second method for you to import yo
 Never reveal your Private Key to anyone. If someone has access to your Private Key, they have full access to all the funds. The only application that should ever need access to your Private Key is a wallet.
 {% endhint %}
 
+### Backup Wallet
+
+It is important that you backup your Key File, passphrase, and private key securely. There are two considerations when choosing a backup method:
```
Comment (BlueLahar) [inline buy-store-and-stake-pokt.md:49]: should be "important that you back up..." (verb is 2 words)

**Comment by jacklaing** [comment]
Credit to BlueLahar (zaatar in Discord) for writing the content and Isaac89 and q846hy for editing.

**Pull request merged**